demos/fab.html

```diff
@@ -39,12 +43,57 @@
           right: 5rem;
         }
       }
+
+      fieldset legend {
+        display: block;
+        padding: 16px;
+        padding-top: 64px;
+        padding-bottom: 24px;
+      }
+
+      .hero button {
+        margin-left: 32px;
+        margin-right: 32px;
+
+      }
+
+      /* Override colors to match catalog theme. */
+
+      .mdc-fab {
```

Comment:
I don't recommend doing this to `.mdc-fab`, as it's misleading as to how the FAB leverage's MDC-Theme (it is supposed to use it's accent color, not it's primary).

If this is the desired style for the hero you can use the [mdc-theme--primary-bg](https://github.com/material-components/material-components-web/tree/master/packages/mdc-theme#theme-color-classes) modifier class to achieve it.